### ISSUE TITLE
Feature Request/Idea: Make embargoes more visible

### ISSUE THREAD

**Issue by DS-INRA** [IQSS/dataverse#9678]
**Overview of the Feature Request**
Make the fact that a file is embargoed visible as restricted files are with an icon

**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
UI depositors (for their own dataset) and guests (when they are browsing for data)

**What inspired the request?**
The fact a file is under embargo is currently not very visible.

**Any brand new behavior do you want to add to Dataverse?**
Make embargoed files pop out in as restricted files do with an icon, maybe something like a clock 🕒 rather than a lock for a clear distinction between embargo/restriction.
Remind the embargo end-date in the info pop-pup of the icon.

**Any open or closed issues related to this feature request?**
None that I found

**Comment by DS-INRA** [comment]
I know UI is nolonger the focus with 6.0 around the corner, but I think it will still be very helpful to keep in mind :)

**Comment by pdurbin** [comment]
I agree it's not super obvious from the file listing page which files are embargoed:

![Screen Shot 2023-07-12 at 2 38 20 PM](https://github.com/IQSS/dataverse/assets/21006/06043240-0f50-4c98-a27a-bad9e5ff67b7)

You have to click:

![Screen Shot 2023-07-12 at 2 38 24 PM](https://github.com/IQSS/dataverse/assets/21006/709c5ac6-5e4f-46de-a185-bb340de2e593)

If it help, you can click here to see if any files are embargoed:

![Screen Shot 2023-07-12 at 2 41 23 PM](https://github.com/IQSS/dataverse/assets/21006/649b6ee3-5b63-49f4-be1a-8ada4b801b90)


At least the search facets show an embargo label:

![Screen Shot 2023-07-12 at 2 36 39 PM](https://github.com/IQSS/dataverse/assets/21006/fd4dbbdd-51f2-4a9d-8489-9096f0d5a2cf)

![Screen Shot 2023-07-12 at 2 36 49 PM](https://github.com/IQSS/dataverse/assets/21006/7f204b1e-f742-4eab-8835-19b9df1ba18c)

The file landing page as well:

![Screen Shot 2023-07-12 at 2 40 23 PM](https://github.com/IQSS/dataverse/assets/21006/0a1387f6-ba07-40bd-a407-893e5da48f94)

**Comment by jo-pol** [comment]
I noticed visibility is implemented on the dataset page in v6.9 as shown below. Could not log in on https://beta.dataverse.org/modern/collections with my credentials for https://demo.dataverse.org/ to compare functionality.

<img width="1140" height="350" alt="Image" src="https://github.com/user-attachments/assets/1f6cbd71-baa2-4941-a503-2a4c72121d0e" />

**Comment by jo-pol** [comment]
By the way, while the files are still in draft, the embargo/retention notices show that fact. For other files the draft status is not shown. That draft tag is convenient when adding files to a previously published dataset.
